### ISSUE TITLE
DateHelper::toExcel() doesn't handle minutes and seconds correctly

### ISSUE THREAD

**Issue by garethellis36** [openspout/openspout#49]
```php
<?php

$dates = [
    "2022-03-22 13:00:00",
    "2022-03-22 13:05:00",
    "2022-03-22 13:10:00",
    "2022-03-22 13:12:14",
];

foreach ($dates as $date) {
    echo "{$date}: " . DateHelper::toExcel(new \DateTimeImmutable($date)) . PHP_EOL;
}
```

Output:
```
2022-03-22 13:00:00: 44642.541666667
2022-03-22 13:05:00: 44642.541666667
2022-03-22 13:10:00: 44642.541666667
2022-03-22 13:12:14: 44642.541666667
```

The outputted values are identical, there is no difference to account for the minutes or seconds.

This has the side-effect that any `DateTimeImmutable` passed as the value of a `Cell` is rendered incorrectly in Excel. I only noticed this because I added custom formatting that would display the minutes.

**Comment by Slamdunk** [comment]
Fixed in https://github.com/openspout/openspout/pull/50

**Comment by Slamdunk** [comment]
Released in `v3.7.3`, thank you

**Comment by garethellis36** [comment]
Wow that was fast! Thanks.
